### PR TITLE
Support rx5700 series issue 134

### PR DESCRIPTION
**Pull request by fboldog**
This PR fixing issue #134.

### CODE REVIEW

File: LibreHardwareMonitorLib/Hardware/Gpu/AmdGpu.cs [hunk 1]
```diff
@@ -44,20 +46,24 @@ public AmdGpu(string name, int adapterIndex, int busNumber, int deviceNumber, IS
...
             _temperatureVddc = new Sensor("GPU VR VDDC", 2, SensorType.Temperature, this, settings);
             _temperatureMvdd = new Sensor("GPU VR MVDD", 3, SensorType.Temperature, this, settings);
             _temperatureLiquid = new Sensor("GPU Liquid", 4, SensorType.Temperature, this, settings);
             _temperaturePlx = new Sensor("GPU PLX", 5, SensorType.Temperature, this, settings);
             _temperatureHotSpot = new Sensor("GPU Hot Spot", 6, SensorType.Temperature, this, settings);
 
             _coreClock = new Sensor("GPU Core", 0, SensorType.Clock, this, settings);
-            _memoryClock = new Sensor("GPU Memory", 1, SensorType.Clock, this, settings);
-
+            _socClock = new Sensor("GPU SOC", 1, SensorType.Clock, this, settings);
```
Comment (Phyxion) [inline LibreHardwareMonitorLib/Hardware/Gpu/AmdGpu.cs]: SoC

File: LibreHardwareMonitorLib/Hardware/Gpu/AmdGpu.cs [hunk 2]
```diff
@@ -44,20 +46,24 @@ public AmdGpu(string name, int adapterIndex, int busNumber, int deviceNumber, IS
...
+            _memoryClock = new Sensor("GPU Memory", 2, SensorType.Clock, this, settings);
+            
             _fan = new Sensor("GPU Fan", 0, SensorType.Fan, this, settings);
             
             _coreVoltage = new Sensor("GPU Core", 0, SensorType.Voltage, this, settings);
+            _memoryVoltage = new Sensor("Memory", 1, SensorType.Voltage, this, settings);
```
Comment (Phyxion) [inline LibreHardwareMonitorLib/Hardware/Gpu/AmdGpu.cs]: GPU Memory for consistency

File: LibreHardwareMonitorLib/Hardware/Gpu/AmdGpu.cs [hunk 3]
```diff
@@ -141,202 +147,286 @@ private void SetDefaultFanSpeed()
...
-                    _coreClock.Value = 0.01f * adlpmActivity.EngineClock;
+                    _temperatureMvdd.Value = null;
+                    if (logDataOutput.sensors[(int)AtiAdlxx.ADLSensorType.PMLOG_TEMPERATURE_VRMVDD].value > 0)
+                    {
+                        _temperatureMvdd.Value = logDataOutput.sensors[(int)AtiAdlxx.ADLSensorType.PMLOG_TEMPERATURE_VRMVDD].value;
+                        ActivateSensor(_temperatureMvdd);
+                    }
+
+                    _temperatureLiquid.Value = null;
+                    if(logDataOutput.sensors[(int)AtiAdlxx.ADLSensorType.PMLOG_TEMPERATURE_LIQUID].value > 0)
```
Comment (Phyxion) [inline LibreHardwareMonitorLib/Hardware/Gpu/AmdGpu.cs]: Add space between if(.

File: LibreHardwareMonitorLib/Interop/AtiAdlxx.cs [hunk 4]
```diff
@@ -255,5 +258,55 @@ internal enum ADLODNTemperatureType
             PLX = 6,
             HOTSPOT = 7
         }
+
```
Comment (Phyxion) [inline LibreHardwareMonitorLib/Interop/AtiAdlxx.cs]: One too many new lines.

**Comment by fboldog** [comment]
Thanks for the review @Phyxion. Fixes coming soon.

Actually [two (three)](https://en.wikipedia.org/wiki/List_of_AMD_graphics_processing_units@#Radeon_RX_5000_series) cards exist with Navi GPU: 5700, 5700 XT (and anniversary edition of 5700 XT)
In the issue #134 my branch tested with 5700 XT (by three different cards), i will ask someone in our HW forum with a normal 5700 to test and make some screenshots :)

**Comment by fboldog** [comment]
@Phyxion requested changes has been fixed, but unfortunately i didn't find anyone to test on 5700.
I think, it will work without any problems on normal 5700 cards, because not too much difference between a normal and xt version.

**Pull request merged**